README.md

```diff
@@ -46,26 +48,338 @@ ember install ember-resources
 ## Example
 
 ```js
-import { useFunction } from 'ember-resources';
+import { trackedFunction } from 'ember-resources';
 
 class MyClass {
-  data = useFunction(this, async () => {
-    let response = await fetch('...');
+  @tracked endpoint = 'starships';
+
+  data = trackedFunction(this, async () => {
+    let response = await fetch(`https://swapi.dev/api/${this.endpoint}`);
     let json = await response.json();
-    return json;
+
+    return json.results;
   }),
+
+  get records() {
+    return this.data.value ?? [];
+  }
+}
+```
+```hbs
+{{this.records}}
+```
+
+In this example, `trackedFunction` will make a call to [StarWars API](https://swapi.dev/)
+and if `endpoint` changes from `starships` to `planets`, the `trackedFunction` will
+automatically re-call the StarWars API to fetch the planets.
+
+## Related addons
+
+List of addons that use and wrap `ember-resources` to provide more specific functionality:
+
+- [ember-data-resources](https://github.com/NullVoxPopuli/ember-data-resources) - resources for reactive data fetching with ember-data
+- [ember-array-map-resource](https://github.com/NullVoxPopuli/ember-array-map-resource) - provides a useArrayMap function which returns a resource that reactively maps data per-element, so that when the overall collection is dirtied, only the changed/new/removed elements affect the mapped collection
+- [ember-use-sound](https://github.com/chrismllr/ember-use-sound) - a resource for interacting with audio files
+
+## What is a Resource?
+
+> They are most similar to class-based Ember helpers, but with a more targeted goal overall. Resources are meant to bridge a gap between imperative programming and declarative programming.
+>
+> Ember templates are declarative. When we design a component, like the profile component from our previous example, we are specifying declaratively the HTML that should be rendered. If the data used in the templates ever updates, then Ember will update the rendered output as well, and we don't have to worry about the details. We don't have to tell Ember which specific steps to take, and when - it figures everything out for us.
+
+<div style="width: 100%; text-align: right;">
```

Comment:
It is tiny bit weird how this is rendered without margins around. The next paragraph then sticks to it.

Reply:
Would [this format](https://stackoverflow.com/questions/2002120/citing-the-author-of-a-blockquote-using-markdown-syntax) be better for citing?